src/api/app/views/layouts/webui2/webui.html.haml

```diff
@@ -16,6 +16,12 @@
     - if User.current.try(:rss_token)
       = auto_discovery_link_tag(:rss, user_rss_notifications_url(token: User.current.rss_token.string, format: :rss), { title: 'Notifications' })
     = auto_discovery_link_tag(:rss, latest_updates_feed_path(format: 'rss'), { title: 'Latest updates' })
+
+    - if Rails.env.test?
```

Comment:
Huaaaaa. Test stuff leaking into production code. Don't...

Reply:
The only alternative I see is adding .erb to webui2/webui2.css.scss and then importing one more file in it if Rails.env - then the code won't appear in production (as uncompiled sass won't in general)

Reply:
How about changing the spec? :thinking:

Reply:
to do what? inject javascript on load?

Reply:
Basically every time you reload a page (e.g. by click) is potentially showing animations and just slows down tests.

For reference:
https://github.com/os-autoinst/openQA/blob/master/assets/assetpack.def#L142

Reply:
> How about changing the spec? thinking

Is there any way of changing the spec support to avoid the javascript animations and the CSS transitions?

The main point of this is that we are facing flickering test because some times capybara is too slow clicking on elements that are moving...

Reply:
Something like this:

https://gist.github.com/andyjbas/9962218

?

Reply:
> Is there any way of changing the spec support to avoid the javascript animations and the CSS transitions?

Which spec are we talking about? :-)

Reply:
Every feature spec that can throw an animation... so, potentially any features spec

Reply:
🐥🔫

Reply:
Mit kanonen auf spatzen schiessen

Reply:
> Something like this:
> 
> https://gist.github.com/andyjbas/9962218
> 
> ?

That could even work if we were using poltergeist - which we don't